libs/blocks/sticky-header/sticky-header.js

```diff
@@ -0,0 +1,24 @@
+import { createTag, getConfig } from '../../utils/utils.js';
+
+export default function decorate($block) {
+  const $header = document.querySelector('header');
+  const $div = $block.firstElementChild;
+  const $a = $block.querySelector('a');
+  const $close = createTag('a', { class: 'sticky-header-close' });
+
+  const { miloLibs, codeRoot } = getConfig();
+  const base = miloLibs || codeRoot;
+  const $closeIcon = createTag('img', { class: 'sticky-header-close-icon', src: `${base}/blocks/sticky-header/close.svg` });
```

Comment:
The close icon is currently too small on mobile devices. Can you look into that? 
![image](https://user-images.githubusercontent.com/115231412/212037350-9f699405-167e-4383-862a-7cd375445ba6.png)